test/remote/gateways/remote_adyen_test.rb

```diff
@@ -140,7 +140,7 @@ def test_successful_purchase_no_cvv
   end
 
   def test_successful_purchase_with_more_options
-    options = @options.merge!(fraudOffset: '1', installments: 2)
+    options = @options.merge!(fraudOffset: '1', installments: 2, shopper_statement: 'statement note', device_fingerprint: 'm7Cmrf++0cW4P6XfF7m/rA')
```

Comment:
The change looks good, I notice that we don't specifically test `telephoneNumber`, but we are testing the other new fields. I just want to make sure that was intentional.

Reply:
It was intention in the sense that it's a common field we don't normally test for anyhow, and now all other tests nominally pass the default phone generated by the `credit_card` creation helper.

Reply:
👍 Works for me.